main.go

```diff
@@ -295,3 +297,18 @@ func buildMetricsServer(m *metricshandler.MetricsHandler, durationObserver prome
 	})
 	return mux
 }
+
+func registerBuildInfo(r prometheus.Registerer) {
+	v := version.GetVersion()
+	promauto.With(r).NewGaugeFunc(prometheus.GaugeOpts{
+		Name: "kube_state_metrics_build_info",
```

Comment:
Do we need it to be `"kube_state_metrics_build_info"`, what about just `build_info`? We have the job label to specify that it pertains to kube-state-metrics right?

Reply:
I prefer the full name `kube_state_metrics_build_info`